seatunnel-e2e/seatunnel-flink-connector-v2-e2e/src/test/java/org/apache/seatunnel/e2e/flink/v2/clickhouse/ClickhouseSourceToConsoleIT.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.e2e.flink.v2.clickhouse;
+
+import org.apache.seatunnel.e2e.flink.FlinkContainer;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.ClickHouseContainer;
+import org.testcontainers.containers.Container;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.lifecycle.Startables;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.stream.Stream;
+
+public class ClickhouseSourceToConsoleIT extends FlinkContainer {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClickhouseSourceToConsoleIT.class);
+    private ClickHouseContainer clickhouse;
+
+    @BeforeEach
+    public void startClickhouseContainer() throws InterruptedException, ClassNotFoundException {
+        clickhouse = new ClickHouseContainer(DockerImageName.parse("yandex/clickhouse-server:22.1.3.7"))
+                .withNetwork(NETWORK)
+                .withNetworkAliases("clickhouse")
+                .withLogConsumer(new Slf4jLogConsumer(LOGGER));
+        //clickhouse.setPortBindings(Lists.newArrayList("8123:8123"));
+        Startables.deepStart(Stream.of(clickhouse)).join();
+        LOGGER.info("Clickhouse container started");
+        Thread.sleep(5000L);
```

Comment:
Use `Awaitility` instead of Thread.sleep ?

example:
```java
Awaitility.given().ignoreExceptions()
.await()
.atMost(5, TimeUnit.SECONDS)
.untilAsserted(() -> initializeClickhouseTable());
```